docs/src/lib/indexing.md

```diff
@@ -100,9 +169,21 @@ In such an operation `AbstractDataFrame` is considered as two-dimensional and `D
     The rule above means that, similar to single-dimensional objects in Base (e.g. vectors),
     `DataFrameRow` is considered to be column-oriented.
 
-If column indexing using `Symbol` names is performed the order of columns in the operation is specified
+Additional rules:
+* in the `df[CartesianIndex(row, col)] .= v`, `df[row, col] .= v` and `df[row, cols] .= v` syntaxes the assignment to `df` is performed in-place;
+* in the `df[rows, col] .= v` and `df[rows, cols] .= v` syntaxes the assignment to `df` is performed in-place;
```

Comment:
Why not merge this with the previous line?

Reply:
The same reason as above - I need it to make sure I do not miss anything when writing tests. Later we can merge them